nengo_bones/templates/setup.cfg.template

```diff
@@ -181,3 +181,10 @@ skip = ./build,*/_build,*-checkpoint.ipynb,./.eggs,./.git,*/_vendor,{{ codespell
 ignore-words-list = {{ codespell.ignore_words | join(",") }}
 {% endif %}
 {% endblock %}
+
+{%- if py_typed %}
+{% block mypy %}
+[mypy]
+python_version = 3.6
```

Comment:
At some point, we might want to add a `min_python_version` variable which we can use to set both this and the `python_requires` in `setup.py`. (I assume that a project should always have this as the minimum version it supports?) This doesn't have to happen in this PR, just something that would be nice for the future.

Reply:
I wouldn't mind doing it now actually because in the projects that use typing we actually have a minimum version of 3.8. So I'll add that in now.

Reply:
This is added now!